### DISCUSSION TITLE
what is  querylog.json json item "Elapsed"   measuring unit??

### DISCUSSION THREAD

**Discussion by andylau004** [AdguardTeam/AdGuardHome#6912] [Q&A]
![image](https://github.com/AdguardTeam/AdGuardHome/assets/2648844/fe0bce2f-dff7-4689-a315-72bc8e66c894)

1.   in querylog.json json file,   "Elapsed"   measuring unit??
Microseconds  ???

so big number

**Comment by cameronj86** [comment]
Shot in the dark, but when I divided by 1,000,000 (meaning `Elapsed` is in nanoseconds), it roughly matches the ms numbers quoted on my dashboard
